src/main/java/aquality/selenium/browser/BrowserFactory.java

```diff
@@ -15,4 +21,22 @@ default IllegalArgumentException getLoggedWrongBrowserNameException() {
     default void logBrowserIsReady(BrowserName browserName) {
         AqualityServices.getLocalizedLogger().info("loc.browser.ready", browserName.toString());
     }
+
+    default <TDriver extends RemoteWebDriver> TDriver getDriver(Class<TDriver> driverClass, Capabilities capabilities) {
```

Comment:
Rename TDriver to T - it is a Code Smell in Java (by Sonar).

Reply:
fixed, but honestly, I think TDriver is better :)